pkg/apis/storage/types.go

```diff
@@ -294,6 +294,23 @@ type CSIDriverSpec struct {
 	// +optional
 	PodInfoOnMount *bool
 
+	// Defines if the underlying volume supports changing ownership and
+	// permission of the volume before being mounted.
+	// If set to Always, SupportsFSGroup indicates that
+	// the volumes provisioned by this CSIDriver support volume ownership and
+	// permission changes, and the filesystem will be modified to match the
+	// defined fsGroup every time the volume is mounted.
+	// If set to Never, then the volume will be mounted without modifying
+	// the volume's ownership or permissions.
+	// Defaults to Evaluate, which results in the volume being examined
+	// and the volume ownership and permissions attempting to be updated
+	// only when the PodSecurityPolicy's fsGroup is explicitly defined, the
+	// fsType is defined, and the PersistentVolumes's accessModes is RWO.
```

Comment:
"accessModes is RWO" -> "accessModes include RWO". It's an array and we want to keep the current behavior:

https://github.com/kubernetes/kubernetes/blob/442a69c3bdf6fe8e525b05887e57d89db1e2f3a5/pkg/volume/csi/csi_util.go#L131-L140

Reply:
I've updated the references to indicate that if accessModes includes RWO then this will pass.